### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by khalp**

### CODE REVIEW

File: TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/includes/ImageHandler.kt [hunk 1]
```diff
@@ -25,31 +28,42 @@ import android.view.ViewConfiguration
...
+    private val rotations: MutableList<Boolean> = mutableListOf()
     private var space = 0f
     private var prevHeight = 0
     private var prevWidth = 0
     private var clickStartTime = 0L
+    private var deleteMode = false
 
     // Create a new ImageView and add it to the container
-    fun addImageToView(uri: Uri) {
+    fun addImageToView(uri: Uri, isRotated: Boolean, x: Float? = null, y: Float? = null, w: Int? = null, h: Int? = null) {
```
Comment (parkerschroeder) [inline TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/includes/ImageHandler.kt:53]: can we remove the extra parameters here? I don't think we're using them now that image upload isn't a thing

File: TwoNote/app/src/main/res/layout/fragment_note_detail.xml [hunk 2]
```diff
@@ -43,11 +43,10 @@
         android:id="@+id/image_container"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/pen_tools"
```
Comment (parkerschroeder) [inline TwoNote/app/src/main/res/layout/fragment_note_detail.xml:46]: probably doesn't matter, but would it make more sense to constrain this to the top of image_tools instead?

File: TwoNote/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -39,4 +38,5 @@
     <string name="action_erase_off">Deactivate erasing</string>
     <string name="confirm_clear_message">Are you sure you want to erase all ink?</string>
     <string name="confirm_clear">Confirm clear</string>
+    <string name="remove_image">Remove images</string>
```
Comment (parkerschroeder) [inline TwoNote/app/src/main/res/values/strings.xml:41]: is this used anywhere?

Comment (khalp) [reply TwoNote/app/src/main/res/values/strings.xml:41]: It's the content description for the image_tools button

**Pull request merged**